pkg/alerts/nrql_conditions_test.go

```diff
@@ -85,6 +112,8 @@ var (
 			"runbook_url": "https://www.example.com/docs"
 		}
 	}`
+
+	entityGuid = common.EntityGUID("NDAwQzA0rEFST1BTfENPTkRJVElPTnw3MzUzNjL")
```

Comment:
Can we namespace this variable name to avoid a naming collision in another test? Maybe `testAlertsEntityGUID`?